unifold/losses/chain_align.py

```diff
@@ -1,183 +1,198 @@
-import torch
-from unifold.data import residue_constants as rc
-from .geometry import kabsch_rmsd, get_optimal_transform, compute_rmsd
-
-
-def multi_chain_perm_align(out, batch, labels, shuffle_times=2):
```

Comment:
I think we could keep this function in case of future uses. you can use a depreciated symbol for the past function.

Reply:
done